### DISCUSSION TITLE
Simulator button just flashes never displays my project

### DISCUSSION THREAD

**Discussion by Cruisechef** [zepp-health/zeppos-forum#382] [Developer Feedback]
I have followed step by step the Zeppos installation several times thinking I have done something wrong but  I cannot get the simulator to connect.  When  I click the Emulator button it just flashes red for a second and nothing ever happens.

I saw a similar post with the same problem and followed the instructions.   I am indeed logged in,  ran the "zues dev" from the hello-world folder.  Still nothing.

![Screenshot 2024-11-02 5 01 05 AM](https://github.com/user-attachments/assets/d3041ac0-7903-4a78-8e40-344c35f27c3c)

**Comment by MyoungXUE** [comment]
Hello, it seems like there might be a permission issue with the User/.zepp directory.

You can try this method.

![image](https://github.com/user-attachments/assets/68bff3b3-8d53-4d8e-a071-38e24bb74cca)

**Comment by winterwolfi** [comment]
ik this is an old thread but this wouldve saved me so much time if i figured this out earlier,

run the command `zeus status` and make sure `simulator host` is set to 127.0.0.1

for some reason it was set to just "h" by default for me. you can set it with `zeus config set simulator_host=127.0.0.1`
